style.css

```diff
@@ -894,6 +894,7 @@ header {
   .navigation {
     background: #3c3a39;
     padding: 20px;
+    height: 120vh;
   }
 
```

Comment:
## Your design
![first (copy)](https://user-images.githubusercontent.com/93039359/181124322-7e74d047-a477-4aa1-97d8-135af7fb4769.png)
## Figma Template
![figma-template](https://user-images.githubusercontent.com/93039359/181124368-cdfa07dc-b939-414a-a0b4-ff6f558501ad.png)

**[Optional]**
- [ ] Your design doesn't contain that burger icon shown in the Figma template. Please add it to your project :blush: